kubecost/templates/aggregator/aggregator-service.yaml

```diff
@@ -16,12 +16,16 @@ spec:
   ports:
     - name: tcp-api
       port: 9004
-      targetPort: 9004
-    {{- if or .Values.saml.enabled .Values.oidc.enabled}}
-    - name: apiserver
+      targetPort: 9004  
+    {{- if or .Values.saml.enabled .Values.oidc.enabled}}    
+    - name: tcp-api-rbac
       port: 9008
       targetPort: 9008
+    {{- else }}
     {{- end }}
+    - name: db-metrics
+      port: 9363
+      targetPort: 9363
```

Comment:
should we wrap this in a conditional?

Reply:
for now, I think we should add the exporter always. up to the user to scrape it or not.
and we may want to merge this with the existing.
and by merge, I mean cherry-pick what we find to be important.